src/main/java/mate/jdbc/model/Car.java

```diff
@@ -0,0 +1,92 @@
+package mate.jdbc.model;
+
+import java.util.List;
+
+public class Car {
+    private Long id;
+    private String model;
+    private Manufacturer manufacturer;
+    private List<Driver> drivers;
+
+    public Car(Long id, String model) {
```

Comment:
This constructor is unnecessary, as you have a constructor with the same parameters on line 34. Please remove this constructor.